README.md

```diff
@@ -1,6 +1,6 @@
 # Simple S3 Resource for [Concourse CI](http://concourse.ci)
 
-Resource to upload files to S3. Unlike the [the official S3 Resource](https://github.com/concourse/s3-resource), this Resource doesn't care about files being versioned.
```

Comment:
I think we got a few lines crossed here. Versioning isn't the issue. The issue is the ability to upload multiple files. Like a lot of the official Concourse `put` resources, `s3-resource` expects to work with a single file, and throws an error if you attempt to run it against multiple files. That's the reason I originally went from the `s3-resource` to uploading via a script.